### DISCUSSION TITLE
readable pdf not searchable by number

### DISCUSSION THREAD

**Discussion by MicheleRoma** [ocrmypdf/OCRmyPDF#1397] [Q&A]
Hello Community,

after OCRmyPDF execution, numbers are not searchable inside the readable pdf:
![Senza titolo](https://github.com/user-attachments/assets/5a9868a6-f547-45cb-a2cb-fc21e8304fe7)

here is how PdfPig read the number:
![Senza titolo2](https://github.com/user-attachments/assets/81e3fd30-acf5-4d52-a742-bb833a8e4615)

is there a workaround to make numbers (ex.  24327) searchable inside pdf and also avoid PdfPig splitting number?

other data inside readable pdf, like name or surname, are fully searchable, but not numbers.

Thanks
Michele

**Comment by jbarlow83** [comment]
Using a current version of ocrmypdf, the text € 24327 is read, according to several programs.
It appears that PdfPig is not capable of layout analysis to determine how to read the text. But I'm not familiar with it.

**Reply by jbarlow83** [reply]
The quick example given in PdfPig's README does not look capable of working out text positions and maybe isn't able to determine where word breaks are.

**Comment by MicheleRoma** [comment]
using latest ocrmypdf dckr image, some currency value are now searchable (eg. €1686). 
I'm going to use IText for text extraction.

thanks
Michele
